### PR TITLE
Replace `Actor_MarkForDeath` with `Actor_Kill`

### PR DESCRIPTION
**Pull request by tom-overton**
A little more than a year ago, we decided on using `Actor_Kill` instead of `Actor_MarkForDeath`: https://discord.com/channels/688807550715560050/873211000122384415/875411525865713724

To say this vote was contentious would be an understatement. However, one thing became *very* clear from the voting; absolutely nobody likes `Actor_MarkForDeath`. There are legitimate criticisms of `Actor_Kill`, but I don't think they should block us from changing the name; perfect should not be made the enemy of good. To keep around `Actor_MarkForDeath` would mean that we would be intentionally differing from our sister project with a name we all dislike for no real reason.

This is mostly a find-and-replace PR, with the only manual change being that I added it to namefixer.py.

### CODE REVIEW

File: src/overlays/actors/ovl_En_Kusa/z_en_kusa.c [hunk 1]
```diff
@@ -470,7 +470,7 @@ void EnKusa_WaitForInteract(EnKusa* this, PlayState* play) {
             }
         }
         if (KUSA_GET_TYPE(&this->actor) == ENKUSA_TYPE_BUSH) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Kusa/z_en_kusa.c:473]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Kusa2/z_en_kusa2.c [hunk 2]
```diff
@@ -878,7 +878,7 @@ void EnKusa2_Init(Actor* thisx, PlayState* play) {
     } else {
         Collider_InitCylinder(play, &this->collider);
         if (!func_80A5BA58(this, play)) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Kusa2/z_en_kusa2.c:881]: Can replace with an early return.

File: src/overlays/actors/ovl_En_M_Fire1/z_en_m_fire1.c [hunk 3]
```diff
@@ -68,7 +68,7 @@ void EnMFire1_Update(Actor* thisx, PlayState* play) {
     s32 pad;
 
     if (Math_StepToF(&this->timer, 1.0f, 0.2f)) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_M_Fire1/z_en_m_fire1.c:71]: Can replace with an early return.

File: src/overlays/actors/ovl_En_M_Thunder/z_en_m_thunder.c [hunk 4]
```diff
@@ -216,9 +216,9 @@ void EnMThunder_Spin_AttackNoMagic(EnMThunder* this, PlayState* play) {
             AudioSfx_PlaySfx(NA_SE_IT_SWORD_SWING_HARD, &player->actor.projectedPos, 4, &gSfxDefaultFreqAndVolScale,
                              &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb);
         }
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
     } else if (!(player->stateFlags1 & PLAYER_STATE1_1000)) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_M_Thunder/z_en_m_thunder.c:221]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Minifrog/z_en_minifrog.c [hunk 5]
```diff
@@ -116,7 +116,7 @@ void EnMinifrog_Init(Actor* thisx, PlayState* play) {
         if ((this->frogIndex == MINIFROG_YELLOW) ||
             ((gSaveContext.save.weekEventReg[isFrogReturnedFlags[this->frogIndex] >> 8] &
               (u8)isFrogReturnedFlags[this->frogIndex]))) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Minifrog/z_en_minifrog.c:119]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Mt_tag/z_en_mt_tag.c [hunk 6]
```diff
@@ -460,7 +460,7 @@ void EnMttag_HandleCantWinChoice(EnMttag* this, PlayState* play) {
             EnMttag_ExitRace(play, TRANS_TYPE_02, TRANS_TYPE_02);
             gSaveContext.eventInf[1] &= (u8)~8;
             gSaveContext.eventInf[1] |= 4;
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Mt_tag/z_en_mt_tag.c:463]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.c [hunk 7]
```diff
@@ -816,7 +816,7 @@ void func_80A6A36C(EnMushi2* this, PlayState* play) {
     }
 
     if (sp20 && (this->unk_36A <= 0)) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Mushi2/z_en_mushi2.c:819]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Pametfrog/z_en_pametfrog.c [hunk 8]
```diff
@@ -205,7 +205,7 @@ void EnPametfrog_Init(Actor* thisx, PlayState* play) {
 
         if (Actor_SpawnAsChild(&play->actorCtx, &this->actor, play, ACTOR_EN_BIGPAMET, this->actor.world.pos.x,
                                this->actor.world.pos.y, this->actor.world.pos.z, 0, 0, 0, 0) == NULL) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Pametfrog/z_en_pametfrog.c:208]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Paper/z_en_paper.c [hunk 9]
```diff
@@ -154,7 +154,7 @@ void EnPaper_Update(Actor* thisx, PlayState* play) {
 
     func_80C1F87C(this);
     if (this->timer == 0) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Paper/z_en_paper.c:157]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c [hunk 10]
```diff
@@ -714,7 +714,7 @@ void EnPoSisters_DeathStage2(EnPoSisters* this, PlayState* play) {
     this->deathTimer++;
 
     if (this->deathTimer == 32) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c:717]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Rr/z_en_rr.c [hunk 11]
```diff
@@ -714,7 +714,7 @@ void func_808FB42C(EnRr* this, PlayState* play) {
             Item_DropCollectible(play, &this->actor.world.pos, ITEM00_SHIELD_HERO);
         }
         Item_DropCollectibleRandom(play, &this->actor, &this->actor.world.pos, 0x90);
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Rr/z_en_rr.c:717]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Sb/z_en_sb.c [hunk 12]
```diff
@@ -368,7 +368,7 @@ void EnSb_Update(Actor* thisx, PlayState* play) {
             this->actor.params = 1;
         }
         Item_DropCollectibleRandom(play, &this->actor, &this->actor.world.pos, 0x80);
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Sb/z_en_sb.c:371]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Sekihi/z_en_sekihi.c [hunk 13]
```diff
@@ -55,7 +55,7 @@ void EnSekihi_Init(Actor* thisx, PlayState* play) {
     s32 pad;
 
     if ((params < SEKIHI_TYPE_0 || params >= SEKIHI_TYPE_MAX) || sOpaDLists[params] == NULL) {
-        Actor_MarkForDeath(&this->dyna.actor);
+        Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Sekihi/z_en_sekihi.c:58]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Snowman/z_en_snowman.c [hunk 14]
```diff
@@ -1103,7 +1103,7 @@ void EnSnowman_UpdateSnowball(Actor* thisx, PlayState* play) {
         }
 
         SoundSource_PlaySfxAtFixedWorldPos(play, &this->actor.world.pos, 20, NA_SE_IT_REFLECTION_WOOD);
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Snowman/z_en_snowman.c:1106]: Can replace with an early return.

File: src/overlays/actors/ovl_En_St/z_en_st.c [hunk 15]
```diff
@@ -839,7 +839,7 @@ void EnSt_Init(Actor* thisx, PlayState* play) {
     this->unk_2C0 = Object_GetIndex(&play->objectCtx, GAMEPLAY_KEEP);
     if (((ENST_GET_3F(&this->actor) != ENST_3F_63) && Flags_GetSwitch(play, ENST_GET_3F(&this->actor))) ||
         (this->unk_2C0 < 0)) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_St/z_en_st.c:842]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Thiefbird/z_en_thiefbird.c [hunk 16]
```diff
@@ -166,9 +166,9 @@ void EnThiefbird_Init(Actor* thisx, PlayState* play) {
     if (this->actor.params == 1) {
         D_80C1392C = 1;
         Math_Vec3f_Copy(&D_80C13920, &this->actor.world.pos);
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
     } else if (STOLEN_ITEM_1 != STOLEN_ITEM_NONE) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Thiefbird/z_en_thiefbird.c:171]: Can replace with early returns.

File: src/overlays/actors/ovl_En_Tite/z_en_tite.c [hunk 17]
```diff
@@ -668,7 +668,7 @@ void func_808951B8(EnTite* this, PlayState* play) {
             func_800B3030(play, &this->limbPos[i], &gZeroVec3f, &gZeroVec3f, 40, 7, 1);
             SoundSource_PlaySfxAtFixedWorldPos(play, &this->limbPos[i], 11, NA_SE_EN_EXTINCT);
         }
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Tite/z_en_tite.c:671]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Tk/z_en_tk.c [hunk 18]
```diff
@@ -928,7 +928,7 @@ void func_80AEE374(EnTk* this, PlayState* play) {
 
     SubS_FindActorCustom(play, &this->actor, NULL, ACTORCAT_NPC, ACTOR_EN_TK, &sp30, func_80AEE300);
     if (sp30.unk_00 == 0) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Tk/z_en_tk.c:931]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Torch2/z_en_torch2.c [hunk 19]
```diff
@@ -153,7 +153,7 @@ void EnTorch2_UpdateDeath(Actor* thisx, PlayState* play) {
 
     // Fall down and become transparent, then delete once invisible
     if (Math_StepToS(&this->alpha, 0, 8)) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Torch2/z_en_torch2.c:156]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Wf/z_en_wf.c [hunk 20]
```diff
@@ -1181,7 +1181,7 @@ void func_80992E0C(EnWf* this, PlayState* play) {
         } else {
             Item_DropCollectibleRandom(play, &this->actor, &this->actor.world.pos, 0x60);
         }
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Wf/z_en_wf.c:1184]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.c [hunk 21]
```diff
@@ -335,7 +335,7 @@ void func_80B32094(EnZoraegg* this, PlayState* play) {
 void func_80B320E0(EnZoraegg* this, PlayState* play) {
     if (Actor_HasParent(&this->actor, play)) {
         Flags_SetSwitch(play, ENZORAEGG_GET_FE00(&this->actor));
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.c:338]: Can replace with an early return.

File: src/overlays/actors/ovl_Item_B_Heart/z_item_b_heart.c [hunk 22]
```diff
@@ -68,7 +68,7 @@ void ItemBHeart_Update(Actor* thisx, PlayState* play) {
     if (!(this->unk_168 < 0.5f)) {
         if (Actor_HasParent(&this->actor, play)) {
             Flags_SetCollectible(play, 0x1F);
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Item_B_Heart/z_item_b_heart.c:71]: Can replace with an early return.

File: src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.c [hunk 23]
```diff
@@ -102,15 +102,15 @@ void ItemEtcetera_WaitForObject(ItemEtcetera* this, PlayState* play) {
 
 void func_8092009C(ItemEtcetera* this, PlayState* play) {
     if (Actor_HasParent(&this->actor, play)) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Item_Etcetera/z_item_etcetera.c:105]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Bean/z_obj_bean.c [hunk 24]
```diff
@@ -378,7 +378,7 @@ void ObjBean_Init(Actor* thisx, PlayState* play) {
         this->dyna.actor.flags |= ACTOR_FLAG_10;
         func_80937C10(this);
         if (!func_80936D58(this, play)) {
-            Actor_MarkForDeath(&this->dyna.actor);
+            Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Bean/z_obj_bean.c:381]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Bean/z_obj_bean.c [hunk 25]
```diff
@@ -714,7 +714,7 @@ void func_80938670(ObjBean* this) {
 
 void func_8093868C(ObjBean* this, PlayState* play) {
     if (this->unk_1B2 <= 0) {
-        Actor_MarkForDeath(&this->dyna.actor);
+        Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Bean/z_obj_bean.c:717]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Danpeilift/z_obj_danpeilift.c [hunk 26]
```diff
@@ -57,7 +57,7 @@ void ObjDanpeilift_Init(Actor* thisx, PlayState* play) {
     DynaPolyActor_Init(&this->dyna, 1);
     DynaPolyActor_LoadMesh(play, &this->dyna, &object_obj_danpeilift_Colheader_000BA0);
     if (this->dyna.bgId == BG_ACTOR_MAX) {
-        Actor_MarkForDeath(&this->dyna.actor);
+        Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Danpeilift/z_obj_danpeilift.c:60]: Can replace with an early return.

File: src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c [hunk 27]
```diff
@@ -164,7 +164,7 @@ void ShotSun_UpdateHyliaSun(ShotSun* this, PlayState* play) {
             }
         }
 
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Shot_Sun/z_shot_sun.c:167]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Ice_Poly/z_obj_ice_poly.c [hunk 28]
```diff
@@ -269,7 +269,7 @@ void func_80931E58(ObjIcePoly* this, PlayState* play) {
         ActorCutscene_StartAndSetUnkLinkFields(this->actor.cutscene, &this->actor);
         if (this->unk_14A == 1) {
             func_80931828(this, play);
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Ice_Poly/z_obj_ice_poly.c:272]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Lift/z_obj_lift.c [hunk 29]
```diff
@@ -102,7 +102,7 @@ void ObjLift_Init(Actor* thisx, PlayState* play) {
     this->dyna.actor.home.rot.z = this->dyna.actor.world.rot.z = this->dyna.actor.shape.rot.z;
     DynaPolyActor_Init(&this->dyna, 1);
     if ((this->unk_178 <= 0) && (Flags_GetSwitch(play, OBJLIFT_GET_7F(&this->dyna.actor)))) {
-        Actor_MarkForDeath(&this->dyna.actor);
+        Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Lift/z_obj_lift.c:105]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Lightblock/z_obj_lightblock.c [hunk 30]
```diff
@@ -93,7 +93,7 @@ void ObjLightblock_Init(Actor* thisx, PlayState* play) {
     DynaPolyActor_Init(&this->dyna, 0);
     Collider_InitCylinder(play, &this->collider);
     if (Flags_GetSwitch(play, LIGHTBLOCK_DESTROYED(&this->dyna.actor))) {
-        Actor_MarkForDeath(&this->dyna.actor);
+        Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Lightblock/z_obj_lightblock.c:96]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Lightblock/z_obj_lightblock.c [hunk 31]
```diff
@@ -167,7 +167,7 @@ void func_80AF3C34(ObjLightblock* this, PlayState* play) {
     if (this->timer <= 0) {
         temp_a0 = this->dyna.actor.cutscene;
         ActorCutscene_Stop(temp_a0);
-        Actor_MarkForDeath(&this->dyna.actor);
+        Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Lightblock/z_obj_lightblock.c:170]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Lightswitch/z_obj_lightswitch.c [hunk 32]
```diff
@@ -315,7 +315,7 @@ void ObjLightSwitch_Fade(ObjLightswitch* this, PlayState* play) {
     this->colorAlpha -= 200;
     ObjLightswitch_SpawnEffects(this, play); // spawn burning fire effect
     if (this->colorAlpha < 0) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Lightswitch/z_obj_lightswitch.c:318]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Roomtimer/z_obj_roomtimer.c [hunk 33]
```diff
@@ -80,7 +80,7 @@ void func_80973DE0(ObjRoomtimer* this, PlayState* play) {
         if (ActorCutscene_GetLength(this->actor.cutscene) != -1) {
             ActorCutscene_StartAndSetUnkLinkFields(this->actor.cutscene, &this->actor);
         }
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Roomtimer/z_obj_roomtimer.c:83]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Taru/z_obj_taru.c [hunk 34]
```diff
@@ -292,14 +292,14 @@ void func_80B9C07C(ObjTaru* this, PlayState* play) {
...
-    Actor_MarkForDeath(&this->dyna.actor);
+    Actor_Kill(&this->dyna.actor);
 }
 
 void func_80B9C1A0(ObjTaru* this, PlayState* play) {
     if (ActorCutscene_GetCanPlayNext(this->dyna.actor.cutscene)) {
         ActorCutscene_StartAndSetUnkLinkFields(this->dyna.actor.cutscene, &this->dyna.actor);
         Flags_SetSwitch(play, OBJ_TARU_GET_7F(&this->dyna.actor));
-        Actor_MarkForDeath(&this->dyna.actor);
+        Actor_Kill(&this->dyna.actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Taru/z_obj_taru.c:302]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c [hunk 35]
```diff
@@ -190,7 +190,7 @@ void ObjTsubo_Init(Actor* thisx, PlayState* play) {
     this->actor.colChkInfo.mass = MASS_IMMOVABLE;
     this->objBankIndex = Object_GetIndex(&play->objectCtx, sPotTypeData[type].objId);
     if (this->objBankIndex < 0) {
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c:193]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c [hunk 36]
```diff
@@ -485,7 +485,7 @@ void func_809289E4(ObjTsubo* this, PlayState* play) {
         func_80927818(this, play, 1);
         SoundSource_PlaySfxAtFixedWorldPos(play, &this->actor.world.pos, 20, NA_SE_EV_POT_BROKEN);
         if (ObjTsubo_IsSceneNotGohtOrTwinmold(this, play)) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c:488]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c [hunk 37]
```diff
@@ -576,7 +576,7 @@ void func_80928F18(ObjTsubo* this, PlayState* play) {
         }
         SoundSource_PlaySfxAtFixedWorldPos(play, &this->actor.world.pos, 20, NA_SE_EV_POT_BROKEN);
         if (ObjTsubo_IsSceneNotGohtOrTwinmold(this, play)) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c:579]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c [hunk 38]
```diff
@@ -590,7 +590,7 @@ void func_80928F18(ObjTsubo* this, PlayState* play) {
         SoundSource_PlaySfxAtFixedWorldPos(play, &this->actor.world.pos, 20, NA_SE_EV_POT_BROKEN);
         SoundSource_PlaySfxAtFixedWorldPos(play, &this->actor.world.pos, 40, NA_SE_EV_DIVE_INTO_WATER_L);
         if (ObjTsubo_IsSceneNotGohtOrTwinmold(this, play)) {
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c:593]: Can replace with an early return.

File: src/overlays/actors/ovl_Obj_Wturn/z_obj_wturn.c [hunk 39]
```diff
@@ -41,7 +41,7 @@ void func_808A7954(ObjWturn* this) {
 void func_808A7968(ObjWturn* this, PlayState* play) {
     if (play->msgCtx.ocarinaMode >= 28 && play->msgCtx.ocarinaMode < 39) {
         Flags_UnsetSwitch(play, this->actor.params);
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
```
Comment (hensldm) [inline src/overlays/actors/ovl_Obj_Wturn/z_obj_wturn.c:44]: Can replace with an early return.

File: src/overlays/actors/ovl_En_Arrow/z_en_arrow.c [hunk 40]
```diff
@@ -382,7 +382,7 @@ void func_8088ACE0(EnArrow* this, PlayState* play) {
             }
             EffectSsStone1_Spawn(play, &this->actor.world.pos, 0);
             SoundSource_PlaySfxAtFixedWorldPos(play, &this->actor.world.pos, 20, sp82);
-            Actor_MarkForDeath(&this->actor);
+            Actor_Kill(&this->actor);
```
[the remark this answers is not in the document]
Comment (tom-overton) [reply src/overlays/actors/ovl_En_Arrow/z_en_arrow.c:385]: Does not match:
image

File: src/overlays/actors/ovl_En_Wf/z_en_wf.c (hunk 20 above)
Comment (tom-overton) [reply src/overlays/actors/ovl_En_Wf/z_en_wf.c:1184]: Does not match:
image

File: src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.c (hunk 21 above)
Comment (tom-overton) [reply src/overlays/actors/ovl_En_Zoraegg/z_en_zoraegg.c:338]: Does not match: 
image

File: src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c (hunk 36 above)
Comment (tom-overton) [reply src/overlays/actors/ovl_Obj_Tsubo/z_obj_tsubo.c:488]: Does not match:
image

File: src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c [hunk 41]
```diff
@@ -734,15 +735,15 @@ void EnPoSisters_SpawnMegClones(EnPoSisters* this, PlayState* play) {
...
-            Actor_MarkForDeath(clone2);
+            Actor_Kill(clone2);
         }
         if (clone3 != NULL) {
-            Actor_MarkForDeath(clone3);
+            Actor_Kill(clone3);
         }
-        Actor_MarkForDeath(&this->actor);
+        Actor_Kill(&this->actor);
     }
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Po_Sisters/z_en_po_sisters.c:747]: kiiiiiiiilll

File: src/overlays/actors/ovl_En_Snowman/z_en_snowman.c [hunk 42]
```diff
@@ -34,8 +34,8 @@ void EnSnowman_SetupDead(EnSnowman* this);
 void EnSnowman_Dead(EnSnowman* this, PlayState* play);
 void EnSnowman_SetupSplitDoNothing(EnSnowman* this);
 void EnSnowman_SplitDoNothing(EnSnowman* this, PlayState* play);
-void EnSnowman_SetupMarkForDeath(EnSnowman* this);
-void EnSnowman_MarkForDeath(EnSnowman* this, PlayState* play);
+void EnSnowman_SetupKill(EnSnowman* this);
+void EnSnowman_Kill(EnSnowman* this, PlayState* play);
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Snowman/z_en_snowman.c:38]: nice catch

**Review by AngheloAlf — APPROVED** [review]
Thanks

File: src/overlays/actors/ovl_En_Snowman/z_en_snowman.c (hunk 42 above)
Comment (tom-overton) [reply src/overlays/actors/ovl_En_Snowman/z_en_snowman.c:38]: After I did a find-and-replace for `Actor_MarkForDeath`, I did a search for `MarkForDeath` by itself specifically to catch stuff like this. I expected to see more than this, though.

File: src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c [hunk 43]
```diff
@@ -215,14 +215,14 @@ void func_80966A68(EnWeatherTag* this, PlayState* play) {
     if ((s16)this->unk154 < 0) { // cast req
         this->unk154 = 0;
         // @BUG redudant code
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c:217]: If this actually is a bug, should be `@bug`.

File: src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c [hunk 44]
```diff
@@ -233,9 +232,10 @@ void DoorShutter_Init(Actor* thisx, PlayState* play2) {
         this->actor.room = -1;
     }
 
-    // clang-format off
-    objId = Object_GetIndex(&play->objectCtx, D_808A2180[sp24].objectId); this->requiredObjBankIndex = objId; if (objId < 0) { Actor_MarkForDeath(&this->actor); return; }
-    // clang-format on
+    if ((this->requiredObjBankIndex = Object_GetIndex(&play->objectCtx, D_808A2180[sp24].objectId)) < 0) {
```
[the remark this answers is not in the document]
Comment (EllipticEllipsis) [reply src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c:235]: Yeah, I don't like it either but I suppose it's better than disabling the formatter. Unless it's just
```c
this->requiredObjBankIndex = Object_GetIndex();
if (this->requiredObjBankIndex < 0) {}
```
or does the actual assignment to `this->requiredObjBankIndex` afterwards like elsewhere?

**Review by hensldm — APPROVED** [review]
Thank you again :)

File: src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c (hunk 44 above)
[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c:235]: Fwiw, everytime I've seen this pattern in effects, it has always been necessary.

File: src/overlays/actors/ovl_Bg_Icicle/z_bg_icicle.c [hunk 45]
```diff
@@ -178,7 +178,7 @@ void BgIcicle_Fall(BgIcicle* this, PlayState* play) {
             func_800C6314(play, &play->colCtx.dyna, this->dyna.bgId);
             this->actionFunc = BgIcicle_Regrow;
         } else {
-            Actor_MarkForDeath(&this->dyna.actor);
+            Actor_Kill(&this->dyna.actor);
             return;
```
[the remark this answers is not in the document]
Comment (tom-overton) [reply src/overlays/actors/ovl_Bg_Icicle/z_bg_icicle.c:182]: Correct, it isn't needed.

File: src/overlays/actors/ovl_En_Snowman/z_en_snowman.c (hunk 42 above)
Comment (tom-overton) [reply src/overlays/actors/ovl_En_Snowman/z_en_snowman.c:38]: They're in Bigslime/Minislime, and I don't feel comfortable enough with those actors to deal with that. I'll leave them for now.

File: src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c [hunk 46]
```diff
@@ -214,7 +214,7 @@ void func_80966A68(EnWeatherTag* this, PlayState* play) {
     }
     if ((s16)this->unk154 < 0) { // cast req
         this->unk154 = 0;
-        // @BUG redudant code
+        // @bug redudant code
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c:217]:
```suggestion
        // @bug redundant code
```

**Review by EllipticEllipsis — APPROVED** [review]

**Pull request merged**